packages/react/src/components/Alert/Alert.tsx

```diff
@@ -136,11 +140,13 @@ class Alert extends AutoControlledComponent<WithAsProp<AlertProps>, AlertState>
     success: PropTypes.bool,
     visible: PropTypes.bool,
     warning: PropTypes.bool,
+    wrapperId: PropTypes.string,
   }
 
   static defaultProps = {
     accessibility: alertBehavior,
     dismissAction: { icon: 'close' },
+    wrapperId: _.uniqueId('alert-wrapper-'),
```

Comment:
This will be the same for all components, ids should be stored under component's state.

Reply:
If its a state and not a prop, then how can I retrieve the value in [behavior](https://github.com/stardust-ui/react/pull/1821/files#diff-4757d822a5e7f14212c430689bfb9d1dR17)?

Reply:
Accessibility behavior receive `state` and `props`, so you will have it ✨